src/utils/validators.js

```diff
@@ -11,37 +11,37 @@ const SPECIAL_CHARACTER_PATTERN = /[`~!@#$%^&*()_|+\-=?;:'",.<>{}[\]\\/]/;
 
 // validators (ordered alphabetically)
 export function code(value) {
-    if (value && value.length > 50) {
+    if (hasValue(value) && value.length > 50) {
```

Comment:
I did some checks and `!!value` returns the same results as the `hasValue` function; i.e. these all produce `false`: `undefined`, `null`, and `''`.

Any reason not to just use `!!value` and drop the `hasValue` function?

One potential upside for the `hasValue` function is that it is testable, but I don't see a test for it. :)

Reply:
The reason I introduced the `hasValue` function, is for numerical variables with value 0. This is a valid value, but would evaluate to false using `!!value` or `Boolean(value)`.

I have had a nasty experience in the past when an api response changed slightly and started returning `0` instead of `"0"`. I then had to go through a lot of the code base and replace a lot of `!!value` checks by something like what I have implemented here. I guess I thought I would be smart and prevent this from happening again.... 

That being said... I actually don't think that any of the form-fields, currently in the user-app, would return a value where `typeof value === 'number'`. Even an `<input type="number" />` returns its value as a string (I hadn't realised this, but just checked).

I guess what I have done here is perhaps a bit over-engineered. Shall I remove the function and replace `hasValue(value)` by `!!value`?

Reply:
I'm OK with it as long as there is a reason for it. :)